### PR TITLE
NLog + log4net NetStandard Support

### LINKED ISSUE #200: Upgrade NLog to 4.5 RTM for .NET Core support
We are currently using NLog 4.4.0-beta13 to get .NET Core support, we need to upgrade to an RTM version once it is released.

### LINKED ISSUE #201: Support log4net integration on .NET Core
This will miss the cut for v4.0 because log4net still doesn't support .NET Core: https://github.com/apache/log4net/pull/16

### PR DESCRIPTION
**Pull request by snakefoot**
Resolves #200 + Resolves #201

### CODE REVIEW

File: src/Castle.Services.Logging.log4netIntegration/ExtendedLog4netFactory.cs [hunk 1]
```diff
@@ -16,14 +16,22 @@ namespace Castle.Services.Logging.Log4netIntegration
...
 
 	using Castle.Core.Logging;
 
 	using log4net;
 	using log4net.Config;
 
+
 	public class ExtendedLog4netFactory : AbstractExtendedLoggerFactory
 	{
+#if NET35 || NET40
```
Comment (jonorossi) [inline src/Castle.Services.Logging.log4netIntegration/ExtendedLog4netFactory.cs]: Can you change this to `#if FEATURE_LEGACY_REFLECTION_API`.

Comment (snakefoot) [reply src/Castle.Services.Logging.log4netIntegration/ExtendedLog4netFactory.cs]: Fixed

**Comment by snakefoot** [comment]
> @jonorossi Add an enhancement to our release notes file in the root of the repo.

Done

**Comment by jonorossi** [comment]
Looks great, merged!

**Pull request merged**